prem_utils/connectors/prem.py

```diff
@@ -55,16 +55,15 @@ def __init__(self, api_key: str, prompt_template: str | None = None) -> None:
         )
 
         self.model_list = [
```

Comment:
I think we can remove this list :)

Reply:
This was mainly for the assert statement, but okay we can remove this. I am doing this in the PR #53

Reply:
Done